src/EnumSet.php

```diff
@@ -54,19 +60,17 @@ public function __construct($enumeration)
                 __NAMESPACE__ . '\Enum'
             ));
         }
+        if (version_compare(phpversion(), '5.6.3', '<')) {
+            $this->integerSize = 4*8;
+        }else {
+            $this->integerSize = PHP_INT_SIZE*8;
+        }
```

Comment:
the problem is that php `pack` only support 32bit in before 5.6
https://php.net/manual/en/function.pack.php